### ISSUE TITLE
[Bug] [A11y] QAB handle contrast and touch target

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#2503]
## Steps to reproduce
Run the Google a11y scanner on a tab, with the QAB handle visible.

### Expected behavior
According to the a11y scanner app:

* touch target should be larger to avoid tapping on other items
* higher foreground-background contrast ratio 
    1. Actual behavior
* touch target: 12dp vs 48dp suggested by the A11y scanner. You either accidentally tap the URL bar (to open) or the bookmark button (to close it). Swiping up/down still works fine though.
* foreground-background contrast ratio: 1.36 vs 3.00 suggested
    1. Device information
* Android device: Note 8 (Android 9)
* Fenix version: 5/14

**Comment by ekager** [comment]
Closing because we have removed the QAB
